packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -321,6 +325,23 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
     }
 
     async $activateByEvent(activationEvent: string): Promise<void> {
+        if (activationEvent.endsWith(':*')) {
+            const baseEvent = activationEvent.substring(0, activationEvent.lastIndexOf(':'));
```

Comment:
Since we know `activationEvent` ends with `:*` we don't need `lastIndexOf`:

```ts
const baseEvent = activationEvent.substring(0, activationEvent.length - 2);
```